### ISSUE TITLE
Error in the displayed code for the Client Side

### ISSUE THREAD

**Issue by ramchandran84** [dotnet/docs#5494]
Hi,

The said code shows WSHTTPBinding but the endpoint address is "net.tcp://...". See below
"
// Create the binding.
WSHttpBinding myBinding = new WSHttpBinding();
myBinding.Security.Mode = SecurityMode.Message;
myBinding.Security.Message.ClientCredentialType =
    MessageCredentialType.Windows;

// Create the endpoint address. 
EndpointAddress ea = new
    EndpointAddress("net.tcp://machineName:8008/Calculator");
"
Regards
Ram

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca217401-155c-a86a-e78d-c03fb2f33835
* Version Independent ID: 8f8200ba-8203-49fe-353b-938408370af9
* Content: [Message Security with a Windows Client](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/message-security-with-a-windows-client)
* Content Source: [docs/framework/wcf/feature-details/message-security-with-a-windows-client.md](https://github.com/dotnet/docs/blob/master/docs/framework/wcf/feature-details/message-security-with-a-windows-client.md)
* Service: **unspecified**
* Product: **dotnet-framework**
* GitHub Login: @BrucePerlerMS
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
@zhenlan can you help with this question?

**Comment by zhenlan** [comment]
Looks like we should just use `NetTcpBinding`. Add @mconnew to confirm.

**Comment by mconnew** [comment]
That's correct, switch to NetTcpBinding

**Comment by Niladri24dutta** [comment]
is there any PR required for this? I can submit a PR if required.

**Comment by rpetrusha** [comment]
If you would like to submit a PR to address this issue, @ramchandran84, that would be great!

**Comment by Niladri24dutta** [comment]
@rpetrusha Can I submit a PR for this ?

**Comment by rpetrusha** [comment]
Sorry, @Niladri24dutta, I'd meant to @-mention you, but didn't. Yes, we'd welcome a PR!

**Comment by Niladri24dutta** [comment]
@rpetrusha ok working on it.

**Comment by Niladri24dutta** [comment]
@rpetrusha as this requires a change in code snippet which is in another repository https://github.com/dotnet/samples/  , so should I fork and create the PR in that repository instead of here? I will mention the issue # in the PR as per the rule.

**Comment by rpetrusha** [comment]
Yes, @Niladri24dutta, fork the samples repository and open the PR there. We're used this repo as the centralized repository for issues for dotnet/docs, dotnet/dotnet-api-docs, and dotnet/samples.

**Comment by Niladri24dutta** [comment]
@rpetrusha I have updated the csharp version of the code snippet but there are some ambiguity in the endpoint declaration for the visual basic version. 

- The endpoint is declared like this in VB version 
`Dim netTcpUri As New Uri("net.tcp://localhost:8006/Calculator")` for the service.But the endpoint has a different port in the web.config 
`<endpoint address="net.tcp://localhost:8008/Calculator"  
                  binding="netTcpBinding"  
                  bindingConfiguration="Windows"  
                  name="WindowsOverMessage"  
                  contract="ServiceModel.ICalculator" /> `

- The client code in VB version uses `WSHttpBinding` and the endpoint URL starts with `http` 
    Dim myBinding As New WSHttpBinding()

    myBinding.Security.Mode = SecurityMode.Message

    myBinding.Security.Message.ClientCredentialType = MessageCredentialType.Windows

    Dim ea As New EndpointAddress("http://localhost:90/Calculator")`

- The web.config (for client) is also different for VB version than the actual endpoint 

   `<client>  
      <endpoint address="net.tcp://machineName:8008/Calculator"   
                binding="netTcpBinding"  
                bindingConfiguration="NetTcpBinding_ICalculator"  
                contract="ICalculator"  
                name="NetTcpBinding_ICalculator">          
      </endpoint>  
    </client>` 

can you confirm that should I make the changes in the VB version too?

**Comment by rpetrusha** [comment]
@Niladri24dutta, I suspect that `localhost:8006` is a typo; it should be localhost:8008` to be consistent both with the configuration file and the C# example. 

Overall, the best approach when having multi-language example is to produce a working example in one language (here, C#) and then convert it to VB. So the examples should correspond unless there's a compelling reason for them not to -- which I don't see in this case.

**Comment by Niladri24dutta** [comment]
@rpetrusha Ok , so I am making the changes in VB version too before making the PR.

**Comment by Niladri24dutta** [comment]
@rpetrusha I have created PR https://github.com/dotnet/samples/pull/95  for this. Can you please review it?
